frontends/p4/validateParsedProgram.cpp

```diff
@@ -48,6 +48,18 @@ void ValidateParsedProgram::postorder(const IR::Method* m) {
     }
 }
 
+/// Check that verify is not used outside of parsers
+void ValidateParsedProgram::postorder(const IR::MethodCallExpression* e) {
```

Comment:
Whoops, actually I don't understand.

* The check is already in IR::MethodCallExpression. I extended the visitor to override that case. Are you suggesting moving it elsewhere? Or keeping it there?

* I do see that using `MethodInstance` would streamline the analysis but I don't see any uses in this pass. Which is the call you were referring to (maybe a link to a line number?)